web/styles/modal.css

```diff
@@ -20,7 +20,7 @@
 .modal__container {
     display: flex;
     flex-direction: column;
-    background-color: hsl(0deg 0% 100%);
```

Comment:
Using the variable changes this to 98% instead of 100% but that's such a small difference that I don't think it'll have much visual effect and it seems good to use the variable instead and keep things consistent.

Reply:
Yeah that seems likely not an intentional difference.

Reply:
Actually, trying out this change, I think 

Here's 100%:

![image](https://github.com/zulip/zulip/assets/2746074/7dd130d8-b621-4adc-a2e7-e1c5e9a886ed)

Here's 98%:

![image](https://github.com/zulip/zulip/assets/2746074/2817b194-f372-4721-8ff5-282668bfa826)

I think that change is visually perceptible, but that the 98% version looks better, so this seems like a good change.